runtime/compiler/env/VMJ9Server.hpp

```diff
@@ -54,6 +54,7 @@ class TR_J9ServerVM: public TR_J9VM
    virtual bool needRelocationsForBodyInfoData() override                { return true; }
    virtual bool needRelocationsForPersistentInfoData() override          { return true; }
    virtual bool needRelocationsForLookupEvaluationData() override        { return true; }
+   virtual bool needRelocationsForStatics() override                     { return true; }
```

Comment:
you meant `needRelocationsForCurrentMethodPC` here I think?

Reply:
Oops! Fixed in: b5c4f331a1